client/app/states/marketplace/details/details.state.js

```diff
@@ -113,11 +113,14 @@
       });
 
       ShoppingCart.add({
+        name: vm.serviceTemplate.name,
+
         dialogUrl: dialogUrl,
         serviceTemplateId: serviceTemplate.id,
+
         data: {
           action: 'order',
-          resource: dialogFieldData,
+          resource: _.cloneDeep(dialogFieldData),
         },
```

Comment:
@himdel Would it be possible to inject `lodash` and call `cloneDeep` via `lodash`?
I believe that is what is done in other places, so just to keep things uniform...